components/SingleArticle.jsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticle } from "../utils/api";
+import { Loading } from "./Loading";
+import { CommentsList } from "./CommentsList";
+import moment from "moment";
+
+export const SingleArticle = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
I would consider setting this to false initially and then set it to true at the beginning of the useEffect